test/subspace-test-client/src/domain_chain_spec.rs

```diff
@@ -89,12 +88,12 @@ pub fn testnet_evm_genesis() -> RuntimeGenesisConfig {
                 .collect(),
             ..Default::default()
         },
-        ethereum: Default::default(),
-        base_fee: Default::default(),
         self_domain_id: evm_domain_test_runtime::SelfDomainIdConfig {
```

Comment:
why not just default instead ?

Reply:
the `domain_id` is set to `Some(DomainId::new(123))` for testing purpose